data/barcode_platemap.csv

```diff
@@ -0,0 +1,10 @@
+Assay_Plate_Barcode,Plate_Map_Name
```

Comment:
this if for tests, correct?

If so, this file should probably live in a different directory (`test/` prefix somewhere), and, either way, it should probably be called something different to avoid user confusion.

Reply:
The data folder was used as an “input” for the Snakefile to read all the files. I am still unsure how the “inputs” should be designed, therefore I just left it there for an open discussion

Reply:
i see. Since we're planning on making `CytoPipe` a python package, we probably don't want to include any experiment specific data in the repo to avoid confusion.

Reply:
Yes. Currently the `snakemake` searches for an input in order to function properly. I'll be removing this folder in order to avoid confusion. 

data folder removed on this commit: d74575671a20747cfe0ac21744b162584cc915de